components/kyma-environment-broker/internal/orchestration/handlers/kyma_handler.go

```diff
@@ -101,6 +119,50 @@ func (h *kymaHandler) validateTarget(spec orchestration.TargetSpec) error {
 	return nil
 }
 
+// ValidateKymaVersion validates provided version. Supports three types of versioning:
+// semantic version, PR-<number>, and <branch name>-<commit hash>.
+// Validates version iff GitHub responded with 4xx code. If GitHub API does not work
+// (e.g. due to API RATE limit), returns version as valid.
```

Comment:
Is this comment correct? From https://docs.github.com/en/rest/overview/resources-in-the-rest-api#rate-limiting I can see that after exceeding rate limit the GitHub responds with 403 so currently `shouldHandle(resp)` function will return error at 159 line.

Reply:
Good observation @polskikiel, I will patch it up in a bit. Thanks!